digdag-cli/src/main/java/io/digdag/cli/GraphvizWorkflowVisualizer.java

```diff
@@ -13,7 +13,7 @@
 import java.io.IOException;
 import java.io.ByteArrayOutputStream;
 import java.io.File;
-import com.google.common.base.Throwables;
+import io.digdag.commons.guava.ThrowablesUtil;
```

Comment:
How about just `io.digdag.commons.ThrowablesUtil` since we're hiding the explicit dependency to Guava ?

Reply:
I don't have strong opinion about the package name. 
My thought is simply we may have some different functionalities under `io.digdag.commons`  in the future.
But currently only this Class exists. 
So I agree with you and change the package name to `io.digdag.commons';

Reply:
Fixed.